applications/StructuralMechanicsApplication/custom_utilities/formfinding_IO_utility.cpp

```diff
@@ -0,0 +1,94 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics 
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher 
```

Comment:
I don't remember writing this :)

Reply:
hahaha, I'll take the credits for that ;)